test_runner/fixtures/neon_fixtures.py

```diff
@@ -874,10 +875,34 @@ def get_pageserver_version(self) -> str:
 
     @cached_property
     def auth_keys(self) -> AuthKeys:
-        pub = (Path(self.repo_dir) / "auth_public_key.pem").read_text()
+        pem_path = self.auth_public_key_path or (Path(self.repo_dir) / "auth_public_key.pem")
+        pub = pem_path.read_text()
```

Comment:
Is `AuthKeys.pub` field actually used for anything?

Reply:
I've removed the public key part of `AuthKeys`.